packages/target-tools/delivery-api-client/index.js

```diff
@@ -1886,8 +1886,11 @@ class DeliveryApi extends BaseAPI {
         });
         return new JSONApiResponse(response, (jsonValue) => DeliveryResponseFromJSON(jsonValue));
     }
-    async execute(imsOrgId, sessionId, deliveryRequest, version) {
-        const response = await this.executeRaw({ imsOrgId: imsOrgId, sessionId: sessionId, deliveryRequest: deliveryRequest, version: version });
+    async execute(imsOrgId, sessionId, deliveryRequest, version, telemetryProvider) {
+        const response = await this.executeRaw({ imsOrgId: imsOrgId, sessionId: sessionId, deliveryRequest: deliveryRequest, version: version, telemetryProvider: telemetryProvider });
+        
+        telemetryProvider.addEntry({});

```

Comment:
Since this is generated code we shouldn't manually modify it.  Instead let's just wait until this fn returns to target.js and call addEntry() there

Reply:
ooops good shout